modules/farm_fd2_school/src/entrypoints/Vue2/App.vue

```diff
@@ -62,14 +67,17 @@
       >Quantity:</label
     >
     <input
+      v-model="quantitySelect"
```

Comment:
Using `quantitySelect` as the property name here may be a little misleading because this element is not a `select` as is the case for the "Crop" and "Unit".  Maybe just `quantity`?